cmd/flag_ssh.go

```diff
@@ -2,6 +2,13 @@ package cmd
 
 import "github.com/urfave/cli/v2"
 
+var InstanceFlag = &cli.StringFlag{
+	Name: "instance",
+	Usage: `Select a specific instance to execute the command against.
+		This is optional and by default will connect to a random instance.
+		This is only used for Server-based workloads that have VM instances.
```

Comment:
It would probably clarify what this means a bunch by providing some examples. "server-based workloads that have VM instances" is technically correct but very abstract and hard for the reader/user to understand what that means.

Reply:
Thanks, updated.